src/content/defi/index.md

```diff
@@ -0,0 +1,245 @@
+
+It works on the premise that the loan is taken out and paid back within the same transaction. If it can't be paid back, the transaction reverts as if nothing ever happened.
+
+This means a lot of logic must be included in a very bespoke transaction. A simple example might be someone using a flash loan to borrow as much of an asset at one price so they can sell it on a different exchange where the price is higher.
+
+So in a single transaction the following happens:
+
+- You borrow X amount of $asset at $1.00 from exchange A
+- You sell X $asset on exchange B for $1.10
+- You pay back loan to exchange A
+- You keep the profit minus the transaction fee
+
+If exchange B's supply dropped suddenly and the user wasn't able to buy enoughh to cover the original loan, the transaction would simply fail.
+
+### Start saving with crypto {#saving}
+
+You can earn interest on your crypto by lending it. For example, you can lend your Dai to a product like Aave and receive Aave Dai (aDai) which is a token that represents your loaned Dai. Your aDai will increase based on the interest rates and you can see it grow in real-time in your wallet. Your aDai also lets you withdraw the equivalent in Dai at any time.
+
+Etheruem has also seen a new way to save emerge: no-loss lotteries.
+
+No-loss lotteries like PoolTogether are a fun and innovative new way to save money. Buy tickets for a lottery that you can't lose. Only one winner is chosen every week, but your tickets roll over and you can withdraw at any time. The prize pot is generated by the interest gained on all the tickets.
+
+### Swapping tokens {#swaps}
+
+There are lots of tokens on Ethereum. Decentralized exchanges let you trade your tokens while always having control of your assets.
+
+When you use a centralized exchange you have to deposit your assets before the trade and trust them to look after them. While your assets are deposited, they're at risk as centralized exchanges are attractive targets for hackers.
+
+### Trading and prediction markets {#trading}
+
+??? [ Not sure what to write here ]
+
+### Grow your portfolio {#investing}
+
+There are fund management products on Ethereum that will try to grow your portfolio based on a strategy of your choice. This is automatic and doesn't need a human manager taking a cut of your profits.
+
+TokenSets, a popular investment product, tokenises investment strategies and calls them sets. A set represents a portfolio with a strategy. For example, their "ETH 26 Day EMA Crossover" set will aim to accumulate as much ETH as possible by rebalancing the portfolio between ETH and USDC. You don't have to manually handle any portfolio rebalancing. As long as you have the Set token in your wallet, you can withraw the portfolio as individual tokens whenever you like.
+
+### Fund your ideas {#crowdfunding}
```

Comment:
I like this section, though I wonder if we should lead with "crowdfunding" in general & the improvements crypto offers. Quadratic funding is a specific subset of crowdfunding & while extremely innovative I don't think it should be the only topic we mention.

General crowdfunding on Ethereum has significant benefits
- Anyone can issue a crowdsale (not sure how much this should be advertised/cautioned)
- Global pool of capital to draw from (anyone can potentially access)
- Transparent! The issuers can prove how much money is raised & code the criteria for anyone to scrutinize, e.g. if not enough is raised by a certain time, return everyone's money.

Reply:
To me, quadratic funding still might be a somewhat niche segment of crowdfunding & has the potential to scare people off.